data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -757,5 +757,45 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "azoth_sword",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "looks_like": "sword_wood",
+    "name": { "str": "Azoth" },
+    "description": "Azoth.",
+    "material": [ "hardsteel" ],
+    "flags": [ "UNBREAKABLE_MELEE", "NONCONDUCTIVE", "NO_REPAIR", "NO_SALVAGE", "MAGIC_FOCUS", "LIGHT_10", "TRADER_AVOID" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SPIN" ],
+    "weight": "1 g",
+    "volume": "1 ml",
```

Comment:
Not sure about these numbers...

Reply:
Technically its ethereal items, maded by only energy, and all of them shouldnt have any weight. But im still think about it, maybe ask KorGgenT about it is good idea

Reply:
Agreed. (And the shield has weight?)

Reply:
Currently all ethereal items use superalloy as material (it will change) and have normal weights.